src/metabase/analytics/prometheus.clj

```diff
@@ -213,7 +213,10 @@
                        {:description "Number of errors when processing metrics in the metrics adjust middleware."})
    (prometheus/counter :metabase-search/index
                        {:description "Number of entries indexed for search"
-                        :labels      [:model]})])
+                        :labels      [:model]})
+   (prometheus/histogram :metabase-search/index-ms
+                         {:description "Average number of ms a single entry took"
+                          :buckets     [0.5 0.9 0.99]})])
```

Comment:
At first I thought these buckets would be percentiles, but now I realize they are just absolute values.  This seems like a very narrow range for these timings to live in.

Reply:
This is mind-bending, but you are right. Re-did it as a counter.